universe/vikram.hello/index.js

```diff
@@ -0,0 +1,177 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of vikram.hello
+//
+// Copyright 2021 undefined Vikram Nagapudi
+//                          vikramnagapudi2004@gmail.com
+// See LICENSE for details
+"use strict";
+const Tp = require('thingpedia');
+const { ComparisonSubqueryBooleanExpression } = require('thingtalk/dist/ast');
+
+
+const tmdbAccess = "https://api.themoviedb.org/3/";
+const multiSearch = "search/multi?api_key="
+const movieSearch = "search/movie?api_key=";
+const finalSearch = "&language=en-US&page=1";
+const topRated = "movie/top_rated?api_key=";
+const nowPlaying = "movie/now_playing?api_key=";
+
+function printMovie (jsonFile) {
+    let parsedArr = jsonFile.results.map((result) => {
+        ({
+            title: result.original_title,
+            description: result.overview,
+            release_date: result.release_date,
+            rating_score: result.vote_average,
+        })
+    });
+    parsedArr = parsedArr[0,3];
```

Comment:
Wait this is also wrong: there is no multi-index access in JS. This expression will return element 3 (0,3 is a comma-operator so it returns the second element)